### DISCUSSION TITLE
Disabling tuplets

### DISCUSSION THREAD

**Discussion by olin256** [Audiveris/audiveris#735] [Q&A]
The music I'm mostly transcribing almost never contains any tuplets; however, sometimes the transcript is "polluted" with incorrectly recognized triplet signs, usually overlapping with note heads and difficult to delete. Is there a way to disable them altogether?

**Comment by hbitteur** [comment]
The only switch related to tuplets is the support for "implicit tuplets", whereby the engine can decide on its own to insert a tuplet.
But there is nothing yet to prevent the (false) recognition of a glyph as an (explicit) tuplet.

We could add a dedicated switch in the BookParameters dialog.
The switch would be on by default but could be manually modified at sheet/book/default levels.

**Reply by olin256** [reply]
Thanks for the answer. Perhaps a systematic solution could be to allow users to set which symbols can be recognized (probably in the SYMBOLS step).
